engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -248,7 +247,7 @@ private void initRenderGraph() {
         renderGraph.addNode(reflectedRefractedClearingNode, "reflectedRefractedClearingNode");
 
         BufferClearingNode readBufferClearingNode = nodeFactory.createInstance(BufferClearingNode.class, DELAY_INIT);
-        readBufferClearingNode.initialise(READ_ONLY_GBUFFER.getConfig(), displayResolutionDependentFBOs,
+        readBufferClearingNode.initialise(displayResolutionDependentFBOs.getFboConfig(new ResourceUrn("engine:sceneOpaque")), displayResolutionDependentFBOs,
```

Comment:
1. ResourceUrn...
2. obtain that FBOconfig only once and then reuse it elsewhere in the method.